.storybook/themes/commerce.scss

```diff
@@ -680,6 +680,46 @@
 						),
 					),
 				),
+				right-rail: (
+					display: grid,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+				),
+				right-rail-full-width-1: (
+					display: none,
+				),
+				right-rail-main: (
+					display: grid,
+					gap: var(--global-spacing-6),
+					max-width: initial,
+					width: calc(100vw - var(--global-spacing-8)),
+					margin: auto,
+				),
+				right-rail-full-width-2: (
+					margin: 0 calc(-1 * var(--global-spacing-5)) 0 0,
+				),
+				right-rail-rail-container: (
+					max-width: initial,
+					width: 100%,
+					margin: auto,
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					gap: var(--global-spacing-5),
+				),
+				right-rail-main-right-rail: (
+					grid-column: initial,
+					padding: initial,
+				),
+				right-rail-main-interior-item: (
+					grid-column: initial,
+					padding: initial,
+					margin: 0 calc(-1 * var(--global-spacing-5)) 0 calc(-1 * var(--global-spacing-5)),
```

Comment:
Why are we using a negative margins?

Reply:
That has been removed

Reply:
@matthewroach I have updated the code base to reflect our discussion in huddle on Monday 7/18